### PR TITLE
Add title to sketch button

### PR DESCRIPTION
**Pull request by daehn**
**What's in this PR?**

* This PR adds the new sketch button design as well as snapshot tests for the `ConfirmAssetViewController`

bildschirmfoto 2016-07-29 um 11 16 29

### CODE REVIEW

**Comment by marcoconti83** [comment]
I like the new sketch button, it's way more discoverable! 👍

File: Wire-iOS/Sources/UserInterface/Components/Controllers/ConfirmAssetViewController.m [hunk 1]
```diff
@@ -81,12 +79,17 @@ - (void)viewDidLoad
...
-    
-    [self updateViewConstraints];
+    [self createConstraints];
     
     [[CASStyler defaultStyler] styleItem:self];
 }
 
+- (void)viewDidLayoutSubviews
+{
+    self.editButton.layer.cornerRadius = CGRectGetHeight(self.editButton.bounds) / 2;
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Components/Controllers/ConfirmAssetViewController.m]: Why is this necessary? Doesn't IconButton update its own cornerRadius?

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Components/Controllers/ConfirmAssetViewController.m]: Should not be necessary, forgot to remove it when I switched to the circular `IconButton`, removed in 6eb05bc

**Comment by daehn** [comment]
@typfel re-review please
